Formula/dxflib.rb

```diff
@@ -21,8 +22,62 @@ class Dxflib < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux: "6ab01712961c1a8561d2fc7926397b75169bb67ef89ebfadf4ee7ead4581d4c3"
   end
 
+  depends_on "qt" => :build
+
+  # Sample DXF file made available under GNU LGPL license.
+  # See https://people.math.sc.edu/Burkardt/data/dxf/dxf.html.
+  resource "testfile" do
+    url "https://people.math.sc.edu/Burkardt/data/dxf/cube.dxf"
+    sha256 "e5744edaa77d1612dec44d1a47adad4aad3d402dbf53ea2aff5a57c34ae9bafa"
+  end
+
   def install
-    system "./configure", "--prefix=#{prefix}"
-    system "make", "install"
+    # For libdxflib.a
+    system "qmake", "dxflib.pro"
+    system "make"
+
+    # Build shared library
+    inreplace "dxflib.pro", "CONFIG += staticlib", "CONFIG += shared"
+    system "qmake", "dxflib.pro"
+    system "make"
+    lib_ext = OS.mac? ? "dylib" : "so"
+
+    (include/"dxflib").install Dir["src/*"]
+    lib.install Dir["*.a", "*.#{lib_ext}"]
```

Comment:
```suggestion

    (include/"dxflib").install Dir["src/*"]
    lib.install Dir["*.a", shared_library("*")]
```
I'll also mention that `Pathname#glob` is probably more meaningful for globbing over files that aren't directories...